.readthedocs.yaml

```
@@ -0,0 +1,16 @@
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  # Build the Ansible docsite
+  commands:
+    - python -m pip install --upgrade --no-cache-dir pip
+    - python -m pip install --exists-action=w --no-cache-dir -r ./requirements.in -c ./requirements.txt
+    - python build.py
```

Can't this output into the right directory by itself? Anyway, it needs to be called under the venv.
```suggestion
    - "${READTHEDOCS_VIRTUALENV_PATH}"/bin/python -I build.py
```